### ISSUE TITLE
False positives antivirus

### ISSUE THREAD

**Issue by amd64fox** [amd64fox/SpotX#75]
Sometimes antiviruses find different threats in `SpotX`, one of the last ones I saw was a certain `Heur.BZC.ONG.Boxter.341.0DC6CB9C`

In this case, the well-known antivirus `BitDefender` for some reason saw a threat in bat files and, what is stranger, he saw a threat only when you throw him files in an archive, and without an archive he could not find anything.

I submitted the full SpotX archive to the BitDefender Malware Research Center.
3 weeks later, I received a response.

<details>
<summary><small>Letter after verification from the BitDefender team</small></summary><p>

![Снимок экрана 2022-07-13 182002](https://user-images.githubusercontent.com/62529699/178770226-a1d155e5-a986-44c4-8a83-c6a996ee6e78.jpg)

</summary><p>
</details>

Summing up, if you are tormented by doubts that there is a virus, trojan or some kind of evil script inside SpotX, then I am always ready to explain any part of the code, even if you are a beginner or, on the contrary, consider yourself an experienced programmer, all code is open for your detailed study.

Have a nice day, everyone.

**Comment by mauriva** [comment]
I use Bitdefender as my AV: it is updated to the last version, but when I try to launch the bat file, it is blocked because he saw an Amsi.Edge.21.9473BD86 threat.
I am on Windows 11 and the bat is running with Powershell.
I don't know how to set the AV to let execute the bat file without blocks...

**Comment by amd64fox** [comment]
Run spotx installation directly via powershell

**Comment by mauriva** [comment]
Thank you for reply.
If I run SpotX directly from powershell I've got the same error...

**Comment by amd64fox** [comment]
Unfortunately, then you need to find a way to create exceptions in your antivirus or just uninstall it and use Microsoft Defender.
